front-end/src/components/Header/Navigation/Navigation.styles.ts

```diff
@@ -1,19 +1,21 @@
-import { createStyles } from '@mantine/core';
-import {
-  MAIN_CHART_COLORS,
-  NAVY,
-  LIME_GREEN,
-  TRANSPARENT_CHART_COLORS,
-} from '../../../Theme/Theme';
+import { createStyles } from "@mantine/core";
+import { TRANSPARENT_CHART_COLORS } from "../../../Theme/Styles";
 
-export const useStyles = createStyles((props) => ({
+export const useStyles = createStyles((theme) => ({
   headerRoot: {
-    backgroundColor: LIME_GREEN,
-    borderBottom: `2px solid ${MAIN_CHART_COLORS[2]}`,
+    backgroundImage:
+      theme.colorScheme === "light"
+        ? theme.fn.linearGradient(1, "#8CB6EA")
+        : theme.fn.linearGradient(300, " #161b22", "#21262d"),
```

Comment:
Remove the line from 
```suggestion
        : theme.fn.linearGradient(300, "#161b22", "#21262d"),
```

Reply:
Done.